typescript/integration-tests/package.json

```diff
@@ -20,7 +20,7 @@
 	},
 	"license": "MIT",
 	"dependencies": {
-		"@itwin/core-bentley": "^3.7.8",
+		"@itwin/core-bentley": "^4.0.0",
 		"@itwin/reality-capture": "workspace:^*",
 		"@itwin/reality-capture-node": "workspace:^*",
 		"@itwin/reality-data-client": "^0.9.0",
```

Comment:
update the reality-data-client across the monorepo to 1.0.0

Reply:
updated reality-data-client and service-authorization